azDevOps/azure/templates/v2/steps/build-audit-dependencies-node.yml

```diff
@@ -8,13 +8,14 @@
 parameters:
   versionSpec: ""
   workingDirectory: ""
+  customRegistry: ""
```

Comment:
Maybe we should default to `useNpmrc`?

Reply:
the default should stay empty 
see updated commit

Reply:
Any reason why we shouldn't supply sane defaults?

Reply:
we can as long as the caller specifying a value of `""` will actually count as a truthy value and not falsey hence defaulting to useNpmrc when in fact there is no npmrc file. 
if you know that an empty string is a truthy value in TFS YAML then let's do it

Reply:
> As an overall point we _really_ need to standardise across one vairable naming scheme... This PR has some with underscores and some camelcase for example.
> 
> Some parameters start with a capital (eurgh) and some don't also... 🤔

It is also my preference to go with the C style snake casing for variable naming convention - but the first iteration was following the MS approach of pascal casing (as in .net) so we can agree on all new code going forward to be snake_cased